src/path_info.cpp

```diff
@@ -151,6 +151,10 @@ std::string PATH_INFO::base_path()
 {
     return base_path_value;
 }
+std::string PATH_INFO::cache_dir()
+{
+    return datadir_value + "cache";
```

Comment:
```suggestion
    return datadir_value + "cache/";
```
For consistency with other paths.